packages/nimble-components/src/table/index.ts

```diff
@@ -687,7 +687,7 @@ export class Table<
                 column.columnInternals
             );
             notifierInternals.subscribe(this);
-            this.columnNotifiers.push(notifier);
+            this.columnNotifiers.push(notifierInternals);
```

Comment:
Note: This isn't functionally required as part of this PR, but I found this mistake when I was writing the column notifiers for the row. Since it was a one-line fix and should be non-controversial, I decided to include it as part of this PR. However, I'd also be fine splitting it out into its own PR if others feel that would be more appropriate.